docs/README.md

```diff
@@ -4,14 +4,29 @@ This repository provides the API requests to provide a tutorial for the Data Exc
 
```

Comment:
I guess that the Introduction should be changed now that this folder doesn't just contain the API related documentation, and it is more like the general documentation.
```
# Tractus-x Umbrella
The Tractus-X Umbrella repository is designed to streamline the management and deployment of theTractus-x Data Space components. It serves as a centralized hub, integrating tools and services like the Portal, CentralIdp, and SharedIdp, while providing the configuration and documentation necessary for setting up and running the Tractus-X environment.
```

Reply:
done